src/lib/poster/ExtraDetails.svelte

```diff
@@ -26,8 +35,8 @@
   }
 </script>
 
-{#if ($page.url?.pathname === "/" || $page.url?.pathname.startsWith("/search/")) && details && dve && dve.length > 0}
```

Comment:
Removing the page check here introduces a bug where:

1. Enable the extra details
2. Go to another users watched list
3. Can see extra details on their page, but can't view <DetailedMenu> to toggle this off/on

I think we could add it back in?

Reply:
I see. I had added it in to make the extra details appear on all lists of movies where I could be interested in the color by status. That's also why on the other hand I then added the line above.
I'll search for a different way to do it

Reply:
Sounds good @AlexPerathoner, do you think we should just support showing ExtraDetails for posters when viewing other users lists? In that case we could just show the ExtraDetails button in nav on that page too, but I'm not sure if that could get confusing for users (if it's showing, it may seem like it's because it is on their list too), what do you think?

Reply:
I personally don't think it would be confusing. After all it's just some extra details on items, not necessarily from their own list. Also with the color they can immediately see if it is indeed on their list

Reply:
@AlexPerathoner That sounds good to me